docassemble/juvenilesealing/data/questions/juvenile-sealing-intro-10.yml

```diff
@@ -181,11 +187,36 @@ subquestion: |
   Once you seal your records, you'll have to unseal them to get
   certified copies. That can sometimes take a while.
 
+  ### What are docket sheets?
+
+  ${docket_sheets_blurb} Docket
+  sheets are papers from the courthouse that say
+  what happened in a case every time you went
+  to court for that case.
+
+  ### Why can sealing be a problem?
```

Comment:
I think this may be a repeat of the info in the intro of the page. Should we consolidate this information into one area in the intro?